api/src/main/java/uk/gov/dhsc/htbhf/claimant/service/payments/PaymentCycleService.java

```diff
@@ -136,4 +141,26 @@ public void updatePaymentCycle(PaymentCycle paymentCycle, PaymentCycleStatus pay
         paymentCycle.setPaymentCycleStatus(paymentCycleStatus);
         paymentCycleRepository.save(paymentCycle);
     }
+
+    /**
+     * Sets the paymentCycle's end date to the start date plus the active cycle duration time.
+     * @param paymentCycle payment cycle to update.
+     */
+    public void updateEndDateForClaimBecomingActive(PaymentCycle paymentCycle) {
```

Comment:
I'm not clear on why this method is necessary? Can't we just amend a PaymentCycle to bring the cycleEndDate forward once we know the claim is PENDING_EXPIRY? Every payment cycle we'll be checking the eligibility - at this point we can reduce the cycle duration and only have to deal with it at one point in the code.

Reply:
Once a claim becomes active we need go reset the payment cycle length, otherwise the next check will occur a week later instead of four weeks.